patches/extra/ungoogled-chromium/remove-uneeded-ui.patch

```diff
@@ -152,6 +153,15 @@
        <template is="dom-if" if="[[!captionSettingsOpensExternally_]]">
 --- a/chrome/browser/resources/settings/about_page/about_page.html
 +++ b/chrome/browser/resources/settings/about_page/about_page.html
+@@ -64,7 +64,7 @@
+         </div>
+ </if>
+         <div class="flex cr-padded-text">
+-<if expr="not chromeos_ash">
++<if expr="False">
```

Comment:
Can confirm that this fixes the extra error message! 🎉

<img width="1912" alt="image" src="https://github.com/ungoogled-software/ungoogled-chromium/assets/72877496/97b71f15-e6c3-486c-b1a3-81e08cc98ba5">

However, there's still an error icon, which I believe is at line 55 of `chrome/browser/resources/settings/about_page/about_page.html` you patched (the other `<if expr="not chromeos_ash">`). Maybe we should also make that `<if expr="False">`?